src/core/import-html/html-to-json.ts

```diff
@@ -111,6 +113,13 @@ const getAttrs = (node: Node) => {
     }
   });
 
+  forEach(attributes, ({ key, value }) => {
```

Comment:
@RafaelGCruvinel The attributes once replaced must be deleted so they dont add up in block json